refactor layouts to be part of graph

mariakleiner opened the pull request:
and delete old nodegraph/ app

sjmiles commented on pkg/Library/Designer/Designer.js:
```diff
@@ -184,15 +182,18 @@ onNodeDelete({eventlet: {key}, graph}, state) {
   if (!node) {
     return this.selectNode(null, state);
   }
+  graph.layout ??= {};
```
I believe these three lines are equivalent to:

`((graph.layout ??= {})[layoutId] ?? = {})[node.id] = value;`

mariakleiner replied:
great, thanks! done

sjmiles commented on pkg/Library/NodeTree/NodeTree.js:
```diff
@@ -155,15 +155,18 @@ async onNodeSelect({eventlet: {key}}) {
   const nodeType = nodeTypes[node.type];
   const hostIds = this.getParticleNames(nodeType).map(particleName => this.hostId(node, particleName));
   await service({kind: 'ComposerService', msg: 'setContainer', data: {hostIds, container}});
+  graph.layout ??= {};
```
As above.

mariakleiner replied:
done, thanks!

sjmiles reviewed (commented):
I love this very much. :heart: :heart: :heart:

P.S. I made a couple notes about syntax for "set this deep object value" which has been an annoyer in JS for years. ECMAScript was working on it, we should have a look-see.

mariakleiner commented:
> I love this very much. ❤️ ❤️ ❤️
> 
> P.S. I made a couple notes about syntax for "set this deep object value" which has been an annoyer in JS for years. ECMA was working on it, we should have a look-see.

Thank you! I agree, it is better now. 
Addressed comments, submitting.
